x/evm/types/params.go

```diff
@@ -107,13 +107,13 @@ func validateBool(i interface{}) error {
 }
 
 func validateEIPs(i interface{}) error {
-	eips, ok := i.([]int)
+	eips, ok := i.([]int64)
 	if !ok {
 		return fmt.Errorf("invalid EIP slice type: %T", i)
 	}
 
 	for _, eip := range eips {
-		if !vm.ValidEip(eip) {
+		if !vm.ValidEip(int(eip)) {
```

Comment:
`int64` here?

Reply:
this is a function that takes an `int` as arg